soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -0,0 +1,2399 @@
+                                                                     { RR_WASTELAND, "Haunted Wasteland" },
+                                                                     { RR_COLOSSUS, "Desert Colossus" },
+                                                                     { RR_MARKET, "Hyrule Market" },
+                                                                     { RR_HC, "Hyrule Castle" },
+                                                                     { RR_KAK, "Kakariko Village" },
+                                                                     { RR_GRAVEYARD, "Graveyard" },
+                                                                     { RR_DMT, "Death Mountain Trail" },
+                                                                     { RR_GC, "Goron City" },
+                                                                     { RR_DMC, "Death Mountain Crater" },
+                                                                     { RR_ZR, "Zora's River" },
+                                                                     { RR_ZD, "Zora's Domain" },
+                                                                     { RR_ZF, "Zora's Fountain" },
+                                                                     { RR_LLR, "Lon Lon Ranch" },
+                                                                     { RR_SFM, "Sacret Forest Meadow" },
+                                                                     { RR_GTG, "Gerudo Training Grounds" },
+                                                                     { RR_DEKU, "Inside the Deku Tree" },
+                                                                     { RR_DC, "Dodongo's Cavern" },
+                                                                     { RR_JABU, "Inside Jabu-Jabu's Belly" },
+                                                                     { RR_FOREST, "Forest Temple" },
+                                                                     { RR_FIRE, "Fire Temple" },
+                                                                     { RR_WATER, "Water Temple" },
+                                                                     { RR_SHADOW, "Shadow Temple" },
+                                                                     { RR_SPIRIT, "Sprit Temple" },
+                                                                     { RR_ICE, "Ice Cavern" },
+                                                                     { RR_BOTW, "Bottom of the Well" },
+                                                                     { RR_TOT, "Temple of Time" },
+                                                                     { RR_OGC, "Outside Ganon's Castle" },
+                                                                     { RR_GANONS_CASTLE, "Ganon's Castle" } };
+
+std::unordered_map<RandomizerCheck, RandomizerRegion> CheckEnumToRegion = {
```

Comment:
instead of one giant `unordered_map` here it'd probably be good to just make an array for each region